modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
+// * observability/network_observability/netobserv-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-netobserv-cli-commands_{context}"]
+= Network Observability CLI reference 
+The Network Observability CLI (`oc netobserv`) is a CLI tool for capturing flow data and packet data for further analysis.
+
+.`oc netobserv` syntax
+[source,terminal]
+----
+$ oc netobserv [<command>] [<feature_command>]<1> [<subcommands>]
+----
+1. Feature commands can only be used with the `oc netobserv flows`. They cannot be used with `oc netobserv packets`.
+[cols="3a,8a",options="header"]
+.Basic commands
+|===
+|Option| Description
+
+| `flows`
+| Capture flows information. For subcommands, see the "Flows subcommands" table on this page. 
+
+| `packets`
+| Capture packets from a specific protocol or port pair, such as 'netobserv packets --filter=tcp,80'. For more information about packet capture, see the "Packet capture subcommand" section on this page.
+
+| `cleanup`
+| Remove netobserv components.
+
+| `version`
+| Print the software version.
+
+| `help`
+| Show help.
+|===
+
+== Feature commands
+The Network Observability Operator features can only be enabled when capturing flows. They do not work with packet capture.
+
+.Syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Network Observability feature enablement commands
+|===
+|Option| Description| Possible values| Default
+
+| `--help`
+| Show help
+| N/A
+| N/A
+
+| `--enable_pktdrop` 	
+| Enable packet drop
+| `true, false`
+| `false`
+
+| `--enable_rtt`
+| Enable round trip time 	
+| `true`, `false` 	
+| `false`
+
+| `--enable_dns`
+| Enable DNS tracking 	
+| `true`, `false` 	
+| false
+
+| `--interfaces`
+| Interfaces to match on the flow e.g., "eth0,eth1" 	
+| ""
+|===
+
+== Flow capture subcommands
+Flow capture has mandatory commands and optional commands. 
+
+.`oc netobserv flows` syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_command>] [<subcommands>]
+----
+
+.Flow capture subcommands
```

Comment:
This table looks funky
<img width="794" alt="Screenshot 2024-05-16 at 11 04 16 AM" src="https://github.com/openshift/openshift-docs/assets/65666095/eccc316d-bb05-4e24-827d-cb270388e448">

Reply:
:/ what noooo I thought I fixed this!! Argh!!

Reply:
Oh okay, I think there were some values without a default, and because I didn't have an empty cell there with a "-" the table was wonk'd out. Do you know if any of those empty defaults _actually_ have defaults @Amoghrd ?

![image](https://github.com/openshift/openshift-docs/assets/77501157/21a0610d-846c-4030-82e9-1addead30aeb)

Reply:
nope they dont have any defaults AFAIK. even protocol does not have any default. @msherif1234 Could you confirm?

Reply:
Yes it used to be tcp but now it has no default the doc in cli repo should be up to date if you to cross reference

Reply:
Ok I adjusted the TCP default